src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+export enum TextError {
+  Add = 'Unable to add a todo',
+  Delete = 'Unable to delete a todo',
+  Update = 'Unable to update a todo',
+}
+
+type Props = {
+  error: boolean;
```

Comment:
you don't need this boolean here! render this component conditionaly by using this boolean in parent component.
```
{isError && <ErrorNotification setError={setError} />}
```